admin/public/src/containers/NamespaceTile.jsx

```diff
@@ -4,35 +4,76 @@ import { Component } from 'react';
 
 export default class NamespaceTile extends Component {
   onClick = () => {
-    const { namespace, selectNamespace } = this.props
-    selectNamespace(namespace.name)
+    const { namespace, selectNamespace, capabilitiesEnabled } = this.props;
+    const { canMakeChanges } = this.state || {};
```

Comment:
Maybe just set state in the constructor so that you don't have to use `|| {}` everywhere?

Reply:
👍